index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
+  </body>
+</html>
```

Comment:
### Your Summary
![pr-summary](https://user-images.githubusercontent.com/93039359/185211439-a72053fe-ac3b-439e-88dd-6fbd406d547c.png)


### Descriptive Summary Example :heavy_check_mark:
![descriptive-summary](https://user-images.githubusercontent.com/93039359/185211469-01a17a39-bb78-44b6-a1a9-39e5100541de.png)


- [ ] You don't have a descriptive summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: